re/postgres/init.go

```diff
@@ -50,6 +50,33 @@ func Migration() *migrate.MemoryMigrationSource {
 					`ALTER TABLE rules DROP COLUMN tags;`,
 				},
 			},
+			{
+				Id: "rules_03",
+				Up: []string{
+					`CREATE TABLE IF NOT EXISTS rules_logs (
```

Comment:
Let's rename to `rule_executions` to better reflect what it does; logs are a little broader term.